modules/document_repository/ajax/documentEditUpload.php

```diff
@@ -54,7 +54,7 @@
     if ($action == 'upload') {
         $puser      = $_POST['user'];
         $category   = $_POST['category'];
-        $site       = $_POST['site'];
+        $site       = $_POST['site'] !== '' ? $_POST['site'] : null;
```

Comment:
similar change should be done for all non-required fields (ie comments, version,...)